references/classification/train.py

```diff
@@ -485,7 +485,10 @@ def get_args_parser(add_help=True):
         "--train-crop-size", default=224, type=int, help="the random crop size used for training (default: 224)"
     )
     parser.add_argument("--clip-grad-norm", default=None, type=float, help="the maximum gradient norm (default None)")
-    parser.add_argument("--ra-sampler", action="store_true", help="whether to use ra_sampler in training")
+    parser.add_argument("--ra-sampler", action="store_true", help="whether to use Repeated Augmentation in training")
+    parser.add_argument(
+        "--ra-reps", default=3, type=int, help="number of repetitions for Repeated Augmentation (default: 3)"
```

Comment:
Thoughts on whether we should have both a boolean and an integer param VS combining the two a single? Other parameters (such as `--clip-grad-norm`, `--auto-augment`) take the latter approach by defining  `default=None` or `default=0` to turn off the feature by default. Though having an extra boolean option is useful when you have many config parameters for the feature, it also creates noise when reading the training log to determine which inputs were actually active for the training.

Reply:
I completely agree, though I followed the style of the EMA flag(s). Should I switch to a more general `--repeated-augmentations` with default 0?

Reply:
Thanks for the input. :) When we introduced EMA, we offered 2 parameters `--model-ema-steps` and `--model-ema-decay` that's why we didnt follow this approach. 

@sallysyw Any thoughts on which approach we should choose here?

Reply:
Makes sense. Two things to consider here:
1. Users may not be aware of what is a "good value" for the number of repetitions. This is solvable though: it could be given as an example in the `help` of the argument.
2. In my fork, I'm adding another boolean flag to keep the same epoch length (rather than cutting it down by the number of repetitions) and replicate the samples within the same minibatch (and GPU), following the original Batch Augmentation paper. If it improves generalization more than the DeiT approach, would you be interested in this as part of another PR?

Reply:
Concerning point 1, I agree with your arguments and I don't have strong opinions. I think your PR is mergeable as-is but I'll let @sallysyw who contributed the class to decide whether we should go with 2 parameters or 1.

Concerning point 2, unless the gap in accuracy is massive, I would keep things simple and leave it as is.

Reply:
Sorry I missed the discussion before. I think for this specific usage case, having 2 parameters for RA are better than 1 because, as @tbennun mentioned, users might don't have a good sense on which `ra_reps` to use. So it's important to have a non-zero default (also for the backward compatibility of replicating ViT training).